### DISCUSSION TITLE
GTD - Refiling items

### DISCUSSION THREAD

**Discussion by levouh** [nvim-neorg/neorg#355] [Q&A] [answered]
As mentioned in `:h neorg-gtd`, clarifying items should involve:

```
      - Move it to the someday/maybe list
      - Make it a reference, removing it from all GTD views and moving it to an external note.
      - Convert it to a project (see {*** Organize})
```

For the former, does that just entail adding a `#context someday` to it or is there a more formal place for "someday" items to get them out of my way? Generically, and noting the comment about references above as well, is there a builtin way to move items from one file to another? E.g. "refile" something from the inbox to a specific file or project?

After looking through `:h neorg-gtd`, as well as the [GTD Wiki](https://github.com/nvim-neorg/neorg/wiki/Getting-Things-Done#Configuration) and the [example workspaces](https://github.com/nvim-neorg/example_workspaces), I do not see a way to do this currently (beyond of course manually moving items between different files).

Is there a "builtin" way to do this?

**Comment by levouh** [comment]
@danymat any ideas? CC-ing you due to you seemingly being the primary author for this feature.

**Reply by esquires** [reply]
Not sure if this solves your problem but [gtd-display-tags](https://github.com/esquires/neorg-gtd-project-tags) would allow you to just change the project tag to "someday" without having to move the task from one file to another.

That said, I am not aware of current logic to suppress particular projects when running the projects view. Personally I like that showing up as something I could look at; I just choose not to look at it very often.

**Comment by danymat** [comment, marked as answer]
> I do not see a way to do this currently (beyond of course manually moving items between different files).
Is there a "builtin" way to do this?

Hello ! it seems that you're referring to the clarify part that is in the plans after the release of Neorg 0.1. In fact, the unclarified views were the start of the broader "Clarify" feature that is coming. The idea will be to create a clarify popup that follows the GTD process, indeed.

**Reply by danymat** [reply]
To complement by previous post:

> For the former, does that just entail adding a #context someday to it or is there a more formal place for "someday" items to get them out of my way?

Adding `#context someday` should clarify the project and put it in someday view, although the someday view currently only shows someday tasks. This addition will be part of a broader addition, that will make project related contexts and due/start dates show up for the project, and not the tasks inside it.

**Reply by levouh** [reply]
Gotcha, so right now it is:

>in the plans after the release of Neorg 0.1

and part of a larger expansion of the "clarify" portion?

**Reply by danymat** [reply]
indeed, although this part:

> Adding #context someday should clarify the project and put it in someday view, although the someday view currently only shows someday tasks. This addition will be part of a broader addition, that will make project related contexts and due/start dates show up for the project, and not the tasks inside it.

Should be done asap
